src/components/calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from './Calculator';
+
+it('calculator snapshot test', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Let us move some tests files inside the test folder that are inside the component.